test/conformance/conformance_async_node.cpp

```diff
@@ -18,158 +18,121 @@
 #pragma warning(disable : 2586) // decorated name length exceeded, name was truncated
 #endif
 
-#include "common/test.h"
-
-#include "common/utils.h"
-#include "common/graph_utils.h"
-
-#include "oneapi/tbb/flow_graph.h"
-#include "oneapi/tbb/task_arena.h"
-#include "oneapi/tbb/global_control.h"
-
 #include "conformance_flowgraph.h"
 
 //! \file conformance_async_node.cpp
 //! \brief Test for [flow_graph.async_node] specification
 
-/*
-TODO: implement missing conformance tests for async_node:
-  - [ ] Write `test_forwarding()'.
-  - [ ] Improve test of the node's copy-constructor.
-  - [ ] Write `test_priority'.
-  - [ ] Rename `test_discarding' to `test_buffering'.
-  - [ ] Write inheritance test.
-  - [ ] Constructor with explicitly passed Policy parameter.
-  - [ ] Concurrency testing of the node: make a loop over possible concurrency levels. It is
-    important to test at least on five values: 1, oneapi::tbb::flow::serial, `max_allowed_parallelism'
-    obtained from `oneapi::tbb::global_control', `oneapi::tbb::flow::unlimited', and, if `max allowed
-    parallelism' is > 2, use something in the middle of the [1, max_allowed_parallelism]
-    interval. Use `utils::ExactConcurrencyLevel' entity (extending it if necessary).
-  - [ ] Write `test_rejecting', where avoid dependency on OS scheduling of the threads; add check
-    that `try_put()' returns `false'
-  - [ ] The `copy_body' function copies altered body (e.g. after successful `try_put()' call).
-  - [ ] The copy constructor and copy assignment are called for the node's input and output types.
-  - [ ] Add CTAD test.
-*/
+using input_msg = conformance::conformance_input_msg<true, true, false>;
+using output_msg = conformance::conformance_output_msg<false, false, false>;
```

Comment:
```suggestion
using input_msg = conformance::conformance_input_msg<true, true, false>;
using output_msg = conformance::conformance_output_msg<false, false, false>;
```
```suggestion
using input_msg = conformance::conformance_input_msg</*default_ctor*/true, /*copy_ctor*/true, /*copy_assign*/false>;
using output_msg = conformance::conformance_output_msg</*default_ctor*/false, /*copy_ctor*/false, /*copy_assign*/false>;
```

What is the key difference between `conformance_input_msg` and `conformance_output_msg`?
Maybe we can just combine them to some `conformance::message` structure?